sdks/java/io/amazon-web-services/build.gradle

```
@@ -39,11 +40,12 @@ dependencies {
   compile library.java.aws_java_sdk_sns
   compile library.java.aws_java_sdk_sqs
   compile library.java.aws_java_sdk_sts
-  compile "commons-lang:commons-lang:2.6"
   compile library.java.jackson_core
   compile library.java.jackson_annotations
   compile library.java.jackson_databind
   compile library.java.slf4j_api
+  compile library.java.joda_time
+  compile "org.apache.httpcomponents:httpcore:4.4.11"
```

library.java.http_core uses version 4.4.12, but I assume 4.4.12 should be compatible with 4.4.11.

```suggestion
  compile library.java.http_core
```